src/objective-c/tests/InteropTests/InteropTests.m

```diff
@@ -1603,7 +1603,13 @@ - (void)testKeepaliveWithV2API {
   GRPCTestRunWithFlakeRepeats(self, ^(GRPCTestWaiter waiterBlock, GRPCTestAssert assertBlock) {
     // The test is highly flaky when ran as part of InteropTestsRemote
     // TODO(jtattermusch): fix and re-enable the test.
-    if ([[self class] isRemoteTest]) {
+    // if ([[self class] isRemoteTest]) {
```

Comment:
Instead of leaving code commented out (it's a smell, since the code could get out of sync if it's not being compiled) ... I'd recommend moving the TODO up to the top of the function, and having an unconditional `return`.

Reply:
Done. Good point.